programmes/migrations/0080_auto_20230627_1038.py

```diff
@@ -0,0 +1,158 @@
+# Generated by Django 4.2.1 on 2023-06-27 08:38
```

Comment:
Je pense qu'on aurait plutôt intérêt à faire ce changement sous forme de script django.  
Car on itère sur pas mal de données (*en mode brutasse*), et ça peut pas mal ralentir la CI ou les tests en local si on teste sur un vrai jeu de données, comme les migrations run à chaque fois qu'on run les tests. 

Si c'est pas bloquant pour déployer le script de le faire hors migration, ça peut être une bonne chose pour le long terme